src/pages/landing/deriv-go/_deriv-go-profits.js

```diff
@@ -36,16 +36,16 @@ const Content = styled(Flex)`
     }
 `
 
-const ImageWrapper = styled.div`
-    width: 100%;
-    margin-right: ${(props) => props.margin_right};
-    @media ${device.tabletL} {
-        margin: 2rem auto;
-    }
-`
-
 const Row = styled(Flex)`
     align-items: center;
+
+    & .content-wrapper {
+        width: 100%;
+        margin-right: ${(props) => props.margin_right};
```

Comment:
The margin_right prop doesn't look like it's being used by the content-wrapper.

Reply:
it's being used to align the images' distance & text